work_queue/src/work_queue_worker.c

```diff
@@ -952,6 +1024,16 @@ static int do_task( struct link *master, int taskid, time_t stoptime )
 				work_queue_task_specify_environment_variable(task,env,value);
 			}
 			free(env);
+		} else if(sscanf(line,"conda_env %s",filename)==1) {
```

Comment:
See above about separating `specify_file` and `conda_env`, that would eliminate the manual fixup here.